machineProblem.py

```diff
@@ -130,6 +131,18 @@ def __init__(self, parent, controller):
         df = pd.read_csv('questionBank.csv')
 
         print(df)
+        
+        with open("questionBank.csv", newline="")as file:
+            reader = csv.reader(file)
+        
+        r=0
+        for col in reader:
+            c=0
+            for row in col:
+                label=tkinter.Label(root, width=10, height=2, text=row, relief=tkinter.RIDGE)
```

Comment:
imagine what "root" means in this line. this is later referred by root.mainloop() in sauce.
think of what is root.mainloop(), you will know where to place this portion of code (inside a class is inappropriate).